### PR TITLE
Alexa UI

### PR DESCRIPTION
**Pull request by aoreilly22**

### CODE REVIEW

**Review by jwoo44 — APPROVED** [review]
Looks good to me

File: src/screens/Home/Home.jsx [hunk 1]
```diff
@@ -60,35 +47,40 @@ export default function Home() {
...
+          <Box w='10'></Box>
           <Input
             placeholder="Name"
             value={name}
             onChange={(event) => setName(event.target.value)}
           />
-          <Text color="white" m="0 1em" fontWeight="bold">
-            Quarter
+          <Box w='20'></Box>
+          <Text color="white">
```
Comment (cvxluo) [inline src/screens/Home/Home.jsx:62]: Text color doesn't seem to work here, at least on preview the text color is still black.

File: src/screens/Home/Home.jsx [hunk 2]
```diff
@@ -111,10 +103,44 @@ export default function Home() {
...
+          </GridItem>
+          <GridItem pl='5' bg='blue.300' area={'abbr'}>
+            <UnorderedList>
+              <ListItem>b/c: because </ListItem>
+              <ListItem>w/: with</ListItem>
+              <ListItem>w/out: without</ListItem>
+              <ListItem>dept: department</ListItem>
+              <ListItem>op: operation</ListItem>
+            </UnorderedList>
+            <Button m="1em" colorScheme='gray'>Add</Button>
```
Comment (cvxluo) [inline src/screens/Home/Home.jsx:141]: Unsure on what this is going to do

**Review by cvxluo — COMMENTED** [review]
Have some miscellaneous concerns, and I'm also generally worried about the overuse of pixel values vs relative.

File: src/screens/Home/Home.jsx (hunk 2 above)
Comment (aoreilly22) [reply src/screens/Home/Home.jsx:141]: The button will be used for when we allow ppl to create custom abbreviations

File: src/screens/Login/Login.jsx [hunk 3]
```diff
@@ -19,13 +21,20 @@ export default function Login() {
...
         </Box>
         <Box align="start">
           <Text fontSize="1xl" fontWeight="bold" color="lightgray">
             Password
           </Text>
-          <Input backgroundColor="lightblue" placeholder="Name" />
+          <InputGroup>
+            <Input backgroundColor="lightblue" placeholder="Password"/>
+            <InputRightElement width='4.5rem'>
+              <Button h='1.75rem' size='sm'>
```
[the remark this answers is not in the document]
Comment (aoreilly22) [reply src/screens/Login/Login.jsx:33]: I was having trouble making it work, so I got rid of the toggle and was planning on implementing that this sprint

File: src/screens/Home/Home.jsx (hunk 1 above)
Comment (aoreilly22) [reply src/screens/Home/Home.jsx:62]: Yes, I'm going to try to resolve that this week

**Pull request merged**